sphinx/source/docs/releases/3.0.0.rst

```diff
@@ -82,3 +82,20 @@ Distance Specs
 
 ``ScreenDistanceSpec`` and ``DataDistanceSpec`` are removed. Use ``SizeSpec``
 instead.
+
+``render_mode``
+...............
+
+``render_mode`` was removed from all annotattions. Annotations were cleanly
+separated between those which render to canvas and those that use HTML and CSS.
+The default is canvas rendering. Use the new ``bokeh.models.annotations.html``
+models whenever previously ``css`` render mode was used. Note that ``Span``
+and ``BoxAnnotation`` don't have a HTML equivalent.
+
+Moves
+~~~~~
+
+Arrow heads
+...........
+
+Arrow heads were moved from ``bokeh.models.arrow_heads`` to ``bokeh.models.annotations.arrows``.
```

Comment:
I don't think this section is necessary. Users should only and ever use `bokeh.models` for importing.